demisto_sdk/commands/doc_reviewer/doc_reviewer.py

```diff
@@ -198,8 +198,11 @@ def run_doc_review(self):
         if len(self.SUPPORTED_FILE_TYPES) == 1:
             click.secho('Running only on release notes', fg='bright_cyan')
 
-        for file_path in self.file_paths:
-            self.get_files_to_run_on(file_path)
+        if self.file_paths:
+            for file_path in self.file_paths:
+                self.get_files_to_run_on(file_path)
+        else:
+            self.get_files_to_run_on()
```

Comment:
1. seems like this part should come before `if self.git_util:`in line 155 
2. self.get_files_to_run_on() may be called twice the way it is now
3. do we intentionally allow running on *both* git files and provided paths at the same time? shouldn't it be mutex?